Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py

```diff
@@ -96,6 +97,23 @@ def _mask(ws, masked_ws, check_if_masked_detectors=True):
         if not check_if_masked_detectors or masked_ws.detectorInfo().hasMaskedDetectors():
             MaskDetectors(Workspace=ws, MaskedWorkspace=masked_ws)
 
+    @staticmethod
+    def _return_numors(paths):
+        regex_all = r'(\*)|(//)|(\+)|(\-)'
```

Comment:
I tried some of the other operators such as * and - in the drill UI. The operator "*" wasn't accepted and it seemed like "-" was interpreted as a range. Are these supposed to result in multiplication and subtraction respectively?
I wonder if the only operator you'll get in here is +?

Reply:
You are absolutely correct, this won't work here. The use of regex for all binary operations in this context is an overkill, as they are simply not accepted by `MultipleFileProperty` validator of `SANSILLReduction` algorithm, and the '-' is going to be interpreted as a range. I simplified it and it will only work with what is accepted: namely '+'. The range given by '-' is interpreted as a sum as well.